File: Lista 2 sala/Python/atividade06.py
import math

A = float(input("Digite o valor de A: "))
B = float(input("Digite o valor de B: "))
C = float(input("Digite o valor de C: "))

delta = B**2 - 4*A*C

if delta > 0:

    raiz1 = (-B + math.sqrt(delta)) / (2*A)
    raiz2 = (-B - math.sqrt(delta)) / (2*A)
    print(f"As raízes da equação são {raiz1:.2f} e {raiz2:.2f}.")
elif delta == 0:
    raiz = -B / (2*A)
    print(f"A equação tem duas raízes: {raiz:.2f}.")
else:
    print("A equação não possui raízes")

File: Lista 2 sala/C/atividade06.c
#include <stdio.h>
#include <math.h>

int main() {
    float A, B, C, delta, raiz1, raiz2, raiz;

    printf("Digite o valor de A: ");
    scanf("%f", &A);

    printf("Digite o valor de B: ");
    scanf("%f", &B);

    printf("Digite o valor de C: ");
    scanf("%f", &C);

    delta = B * B - 4 * A * C;

    if (delta > 0) {
        raiz1 = (-B + sqrt(delta)) / (2 * A);
        raiz2 = (-B - sqrt(delta)) / (2 * A);
        printf("As raizes da equacao sao %.2f e %.2f.\n", raiz1, raiz2);
    } else if (delta == 0) {
        raiz = -B / (2 * A);
        printf("A equacao tem uma raiz: %.2f.\n", raiz);
    } else {
        printf("A equacao nao possui raizes reais.\n");
    }

    return 0;
}
